application/views/admin/globalsettings/_security.php

```diff
@@ -188,15 +194,21 @@
                     <?php eT("Maximum number of attempts:"); ?>
                 </label>
                 <div class="">
-                    <input class="form-control" min="0" type="number" name="maxLoginAttemptParticipants" value="<?= Yii::app()->getConfig('maxLoginAttemptParticipants') ?>" />
+                    <input class="form-control" type="number" min="1" step="1" pattern="^\d*$" name="maxLoginAttemptParticipants" placeholder="<?= gT("Disabled") ?>"
+                        value="<?= App()->getConfig('maxLoginAttemptParticipants') !== "" ??  intval(App()->getConfig('maxLoginAttemptParticipants')) ?>"
+                    />
+                    <div class="form-text"><?= gT("Set an empty value disable brute force protection. Number of attemps are never checked.") ?></div>
                 </div>
             </div>
             <div class="mb-3">
                 <label class="form-label" for='timeOutParticipants'>
                     <?php eT("Lockout time in seconds (after maximum number of attempts):"); ?>
                 </label>
                 <div class="">
-                    <input class="form-control" type="number" min="0" name="timeOutParticipants" value="<?= Yii::app()->getConfig('timeOutParticipants') ?>" />
+                    <input class="form-control" type="number" min="0" step="1" pattern="^\d*$" name="timeOutParticipants" placeholder="<?= gT("Disabled") ?>"
+                        value="<?= App()->getConfig('timeOutParticipants') !== "" ?? intval(App()->getConfig('timeOutParticipants')) ?>"
+                    />
+                    <div class="form-text"><?= gT("Set an empty value or 0 disable brute force protection. Number of attemps are deleted each time.") ?></div>
```

Comment:
Please write this sentence like this:
"Set an empty value or 0 to disable brute force protection. Number of attempts are deleted each time."